hmcl-dev@3.5.3.228: download via aria2 failed

~~~
~ $ scoop update *
hmcl-dev: 3.5.3.227 -> 3.5.3.228
MaaAssistantArknights: 4.8.0-beta.2 -> 4.8.0-beta.3
python: 3.11.0 -> 3.11.1
qbittorrent-enhanced: 4.4.5.10 -> 4.5.0.10
Updating 4 outdated apps:
Updating 'hmcl-dev' (3.5.3.227 -> 3.5.3.228)
Downloading new version
Starting download with aria2 ...
Download: 12/08 23:44:05 [ERROR] CUID#8 - Download aborted. URI=https://maven.aliyun.com/repository/central/org/glavo/hmDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://maven.aliyun.com/repository/central/org/glavo/hmclDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: fb162b|ERR |       0B/s|C:/Users/zhifa/scoop/cache/hmcl-dev#3.5.3.228#https_maven.aliyun.com_repository_central_org_glavo_hmcl_hmcl-dev_3.5.3.228_hmcl-dev-3.5.3.228.exe_HMCL.exe
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-dev/3.5.3.228/hmcl-dev-3.5.3.228.exe#/HMCL.exe
    referer=https://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-dev/3.5.3.228/hmcl-dev-3.5.3.228.exe#/
    dir=C:\Users\zhifa\scoop\cache
    out=hmcl-dev#3.5.3.228#https_maven.aliyun.com_repository_central_org_glavo_hmcl_hmcl-dev_3.5.3.228_hmcl-dev-3.5.3.228.exe_HMCL.exe

ERROR & 'C:\Users\zhifa\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\zhifa\scoop\cache\hmcl-dev.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.3 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=16 --max-connection-per-server=16 --min-split-size=1M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=26704 --continue --summary-interval=0 --auto-save-interval=1 --all-proxy='127.0.0.1:7890'

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/seumsc/scoop-seu/issues/new?title=hmcl-dev%403.5.3.228%3a+download+via+aria2+failed
~~~

--- a/bucket/hmcl-dev.json
+++ b/bucket/hmcl-dev.json
@@ -6,10 +6,10 @@
     "suggest": {
         "JRE with JFX": [
             "java/zulufx8-jre",
-            "java/zulufx16-jre"
+            "java/zulufx17-jre"
         ]
     },
-    "url": "https://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-dev/3.5.3.228/hmcl-dev-3.5.3.228.exe#/HMCL.exe",
+    "url": "https://ci.huangyuhui.net/job/HMCL/228/artifact/HMCL/build/libs/HMCL-3.5.3.228.exe#/HMCL.exe",
     "hash": "sha1:1d8aaca4f895e6d69f7d1d4535bf88b57c5099ee",
     "pre_install": [
         "function ensureFile([String] $name) {",
@@ -35,7 +35,7 @@
         "jsonpath": "$.version"
     },
     "autoupdate": {
-        "url": "https://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-dev/$version/hmcl-dev-$version.exe#/HMCL.exe",
+        "url": "https://ci.huangyuhui.net/job/HMCL/$buildVersion/artifact/HMCL/build/libs/HMCL-$version.exe#/HMCL.exe",
         "hash": {
             "url": "https://hmcl.huangyuhui.net/api/update_link?channel=dev&download_link=true",
             "jsonpath": "$.exesha1"

--- a/bucket/hmcl.json
+++ b/bucket/hmcl.json
@@ -6,10 +6,10 @@
     "suggest": {
         "JRE with JFX": [
             "java/zulufx8-jre",
-            "java/zulufx16-jre"
+            "java/zulufx17-jre"
         ]
     },
-    "url": "http://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-stable/3.5.3/hmcl-stable-3.5.3.exe#/HMCL.exe",
+    "url": "https://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-stable/3.5.3/hmcl-stable-3.5.3.exe#/HMCL.exe",
     "hash": "sha1:71d097d12491a6e0c9e3b3bffaf98065ab322631",
     "pre_install": [
         "function ensureFile([String] $name) {",
@@ -35,7 +35,7 @@
         "jsonpath": "$.version"
     },
     "autoupdate": {
-        "url": "http://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-stable/$version/hmcl-stable-$version.exe#/HMCL.exe",
+        "url": "https://maven.aliyun.com/repository/central/org/glavo/hmcl/hmcl-stable/$version/hmcl-stable-$version.exe#/HMCL.exe",
         "hash": {
             "url": "https://hmcl.huangyuhui.net/api/update_link?channel=stable&download_link=true",
             "jsonpath": "$.exesha1"

